{"name": "test_add_member_fail[user3-1111-请填写合法的手机号]", "status": "passed", "attachments": [{"name": "log", "source": "c2e2c417-6ad4-4f84-9ab0-05f02a550e68-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "user", "value": "'user3'"}, {"name": "phone", "value": "'1111'"}, {"name": "expect", "value": "'请填写合法的手机号'"}], "start": 1626597928471, "stop": 1626597939719, "uuid": "b46f8df8-a361-4aef-876e-80df68f513d5", "historyId": "234428484d32023bba9f8116ebd2db9d", "testCaseId": "b251c483add15ec609094c40b1ea12dc", "fullName": "testcases.test_addMember.TestAddMember#test_add_member_fail", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_addMember"}, {"name": "subSuite", "value": "TestAddMember"}, {"name": "host", "value": "z"}, {"name": "thread", "value": "11180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_addMember"}]}